build.assets/Makefile

```diff
@@ -82,6 +82,8 @@ REQUIRE_HOST_ARCH = $(if $(filter-out $(ARCH),$(RUNTIME_ARCH)),$(error Cannot cr
 # needed on CI as they start with a fresh state each time.
 PULL_ON_CI = $(if $(filter $(CI),true),docker inspect --type=image $(1) >/dev/null 2>&1 || docker pull $(1) || true)
 
+RELEASE_TARGET ?= release
```

Comment:
The comments about release-unix-preserving-webassets etc should probably be attached to this line. It is far from where it is used so should have some comment at least.

Reply:
See 1de16ff8d8b47eb3abcbe89c57729ce328d56fa1